app/Filament/Pages/ReportLibrary.php

```diff
@@ -40,7 +40,7 @@
 
 class ReportLibrary extends Page
 {
-    protected static ?string $navigationGroup = 'Reporting';
+    protected static ?string $navigationGroup = 'Report Centergi';
```

Comment:
```suggestion
    protected static ?string $navigationGroup = 'Report Center';
```

Reply:
Hi @Orrison 
resolved this. sorry by mistake and in hurry i did this mistake. next time will keep this things in mind.